Kitodo/src/main/java/de/sub/goobi/forms/BenutzerverwaltungForm.java

```diff
@@ -224,6 +225,47 @@ public String addToGroup() {
         return null;
     }
 
+    /**
+     * Remove user from client.
+     *
+     * @return empty String
+     */
+    public String deleteFromClient() {
+        int clientId = Integer.parseInt(Helper.getRequestParameter("ID"));
```

Comment:
- Here you are using `Integer.parseInt` to get the numeric `ID` value. In method `addToClient` you are using `Integer.valueOf`. Maybe you can change one to the other? 
- Both methods are throwing a `NumberFormatException ` and this code is in frontend classes, you should handle this.